### PR TITLE
Added the Rowdy Roos voting strategy [rowdy-roos]

### PR DESCRIPTION
**Pull request by npbroo**
Fixes # .

Changes proposed in this pull request:
- 
- 
-

### CODE REVIEW

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Plz revert changes from package-lock and yarn.lock

**Comment by npbroo** [comment]
I have reverted the changes for yarn.lock and package-lock.json please review. Thank you

File: src/strategies/rowdy-roos/index.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import { BigNumber, BigNumberish } from '@ethersproject/bignumber';
+import { Multicaller } from '../../utils';
+import { formatUnits } from '@ethersproject/units';
+
+export const author = 'Kanga';
```
Comment (ChaituVR) [inline src/strategies/rowdy-roos/index.ts:5]:
```suggestion
export const author = 'npbroo';
```

File: src/strategies/rowdy-roos/index.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    stakingPool.call(address, options.staking, 'getStakedTokens', [address]);
+    tokenPool.call(address, options.token, 'balanceOf', [address]);
+  });
+
+  const [stakingResponse, tokenResponse]: [
+    Record<string, number[]>,
+    Record<string, BigNumberish>
+  ] = await Promise.all([stakingPool.execute(), tokenPool.execute()]);
+
+  await addresses.forEach((address) => {
```
Comment (ChaituVR) [inline src/strategies/rowdy-roos/index.ts:45]:
```suggestion
  addresses.forEach((address) => {
```

File: src/strategies/rowdy-roos/index.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    stakingResponse[address].forEach((id) => {
+      stakingPool.call(id, options.staking, 'currentRewardsOf', [id])
+    });
+  });
+
+  const [stakedRewardsResponse]: [
+    Record<string, BigNumberish[]>,
+  ] = await Promise.all([stakingPool.execute()]);
```
Comment (ChaituVR) [inline src/strategies/rowdy-roos/index.ts:53]:
```suggestion
  const stakedRewardsResponse: Record<string, BigNumberish[]> = await stakingPool.execute();
```

File: src/strategies/rowdy-roos/index.ts [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+  return Object.fromEntries(
+    addresses.map((address) => {
+      const claimedCount = parseInt(formatUnits(BigNumber.from(tokenResponse[address]), options.decimals));
+      let total_staked_reward = 0
+      stakingResponse[address].forEach((id) => {
+        total_staked_reward+=parseInt(formatUnits(BigNumber.from(stakedRewardsResponse[id]), options.decimals))
```
Comment (ChaituVR) [inline src/strategies/rowdy-roos/index.ts:60]:
```suggestion
        total_staked_reward += parseInt(formatUnits(BigNumber.from(stakedRewardsResponse[id]), options.decimals))
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**